### ISSUE TITLE
Latest provisioning gem is incompatible with semi-recent Chef

### ISSUE THREAD

**Issue by tylercloke** [chef/chef-provisioning#374]
Install semi-recent `ChefDK`:

```
++> chef --version
Chef Development Kit Version: 0.6.2
chef-client version: 12.3.0
```

Install latest `chef-provisioning` gem into `ChefDK`:

```
++>  chef exec gem install chef-provisioning
```

Now, when you run anything chef, you get a WinRM conflict:

```
++> knife
...
/opt/chefdk/embedded/lib/ruby/site_ruby/2.1.0/rubygems/specification.rb:2104:in `raise_if_conflicts': Unable to activate winrm-s-0.2.4, because winrm-1.3.3 conflicts with winrm (~> 1.2.0) (Gem::ConflictError)
	from /opt/chefdk/embedded/lib/ruby/site_ruby/2.1.0/rubygems/specification.rb:1282:in `activate'
	from /opt/chefdk/embedded/lib/ruby/site_ruby/2.1.0/rubygems/specification.rb:1316:in `block in activate_dependencies'
	from /opt/chefdk/embedded/lib/ruby/site_ruby/2.1.0/rubygems/specification.rb:1302:in `each'
	from /opt/chefdk/embedded/lib/ruby/site_ruby/2.1.0/rubygems/specification.rb:1302:in `activate_dependencies'
	from /opt/chefdk/embedded/lib/ruby/site_ruby/2.1.0/rubygems/specification.rb:1284:in `activate'
	from /opt/chefdk/embedded/lib/ruby/site_ruby/2.1.0/rubygems.rb:198:in `rescue in try_activate'
	from /opt/chefdk/embedded/lib/ruby/site_ruby/2.1.0/rubygems.rb:195:in `try_activate'
	from /opt/chefdk/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:126:in `rescue in require'
	from /opt/chefdk/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:39:in `require'
	from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/knife-windows-0.8.5/lib/chef/knife/bootstrap_windows_base.rb:22:in `<top (required)>'
	from /opt/chefdk/embedded/apps/chef/lib/chef/knife/core/subcommand_loader.rb:44:in `load'
	from /opt/chefdk/embedded/apps/chef/lib/chef/knife/core/subcommand_loader.rb:44:in `block in load_commands'
	from /opt/chefdk/embedded/apps/chef/lib/chef/knife/core/subcommand_loader.rb:44:in `each'
	from /opt/chefdk/embedded/apps/chef/lib/chef/knife/core/subcommand_loader.rb:44:in `load_commands'
	from /opt/chefdk/embedded/apps/chef/lib/chef/knife.rb:127:in `load_commands'
	from /opt/chefdk/embedded/apps/chef/lib/chef/knife.rb:147:in `list_commands'
	from /opt/chefdk/embedded/apps/chef/lib/chef/application/knife.rb:198:in `print_help_and_exit'
	from /opt/chefdk/embedded/apps/chef/lib/chef/application/knife.rb:162:in `validate_and_parse_options'
	from /opt/chefdk/embedded/apps/chef/lib/chef/application/knife.rb:140:in `run'
	from /opt/chefdk/embedded/apps/chef/bin/knife:25:in `<top (required)>'
	from /opt/chefdk/bin/knife:51:in `load'
	from /opt/chefdk/bin/knife:51:in `<main>'
```

**Comment by tylercloke** [comment]
Fixed on `ChefDK` 0.7:

```
++> chef --version
Chef Development Kit Version: 0.7.0.rc.0
chef-client version: 12.4.0
```

```
++> chef exec gem install chef-provisioning
```

```
++> knife
<no errors>
```

**Comment by tylercloke** [comment]
Fixed on `ChefDK` 0.7:

```
++> chef --version
Chef Development Kit Version: 0.7.0.rc.0
chef-client version: 12.4.0
```

```
++> chef exec gem install chef-provisioning
```

```
++> knife
<no errors>
```
